.prettierrc.js

```diff
@@ -1,17 +1,16 @@
 module.exports = {
+  // NOTE: This is actually the default value but it is being overwritten by
+  // the solidity plugin somehow.
   bracketSpacing: true,
-  printWidth: 129,
-  semi: false,
-  trailingComma: "es5",
+  trailingComma: "all",
```

Comment:
We already enable trailing commas in most places (specifically all places supported by ES5, so objects, imports, arrays) and just not in function arguments:
```
foo(
  a,
  b
)
```
becomes:
```
foo(
  a,
  b,
)
```

This is beneficial for the same reason the other currently enabled trailing commas are beneficial. Specifically, the main argument for them is that they improve diffs as adding new parameters only changes one line instead of two.

I imagine one of the reasons `all` is not the default is that it requires [NodeJS >8 to support this](https://prettier.io/docs/en/options.html#trailing-commas). This is a non-issue for us for two reasons:
- We are converting our codebase to TS, and TS should generate compatible code regardless of formatting
- Our current JS code runs in NodeJS LTS (12.0) which already supports it

Reply:
I like this.